salt/config/__init__.py

```diff
@@ -1866,10 +1866,10 @@ def load_config(path, env_var, default_path=None, exit_on_config_errors=True):
 
     opts = {}
 
+    opts['conf_file'] = path
```

Comment:
What will the effect be if the path is not readable?

Reply:
If the path is not readable, it should default to using the configurations from salt.config.__init__

Don't merge this yet, i need to look at the tests.